modules/concept-docs/pages/data-durability-acid-transactions.adoc

```diff
@@ -159,6 +159,52 @@ This durability is enforced by the database,
 // https://docs.couchbase.com/java-sdk/current/concept-docs/durability-replication-failure-considerations.html#older-server-versions
 // Change link once 3.3 is archived?
 
+
+
+
+== Reading from Replicas
+
+By default, read requests from the SDK are directed to the node with the active vbucket.
+This bucket will usually be updated before any replicas (see <#durability,Durability> above),
+and is best for consistency.
+
+Some use cases benefit -- for consistency or availability -- from accessing both the active copy of the document, and/or one or all replicas.
+This can be from the {anyreplica-api}[data service],
+and xref:howtos:subdocument-operations.adoc#reading-sub-documents-from-replicas[SubDoc Read from Replica].
+The method can either be one to return either the first result returned -- active or replica --
+which is useful if a node is timing out;
+or to return _all_ of the results, for client code to handle any inconsistency, or to build a consensus answer.
+
+=== Local Availability Zone Reads
+
+IMPORTANT: Zone Aware Replica Reads are only accessible with the {name-sdk} working with Couchbase Server 7.6.2 or newer (Capella or self-managed).
+
+For Capella users with high data volumes, egress charges for reads from other Availability Zones (AZ) in AWS can be a significant cost.
+The {name-sdk}, when making read replica requests, can be zone aware --
```

Comment:
This is really a matter of configuration. The SDK is really just making a request to a "Preferred Server Group" that happens to be aligned with Zones if the user chooses to use it that way.